MANIFEST.in

```diff
@@ -1,30 +1,93 @@
-include README.rst COPYING MANIFEST.in
+include *.ini
 include *.md
-include config/linkchecker-completion config/create.sql
-include config/linkcheckerrc
-include config/linkchecker.apache2.conf install-rpm.sh
-include linkchecker.freecode
-include cgi-bin/lc.wsgi cgi-bin/README
+include *.rst
+include *.txt
+include COPYING
+include Dockerfile
+include MANIFEST.in
 include Makefile
-include cgi-bin/lconline/*.html cgi-bin/lconline/*.de cgi-bin/lconline/*.en
-include cgi-bin/lconline/*.js cgi-bin/lconline/*.css cgi-bin/lconline/*.ico
-include po/*.po po/*.mo po/*.pot po/Makefile
-include doc/*.example doc/*.md doc/*.txt
-include doc/html/*.ico
-include doc/html/*.html
-include doc/html/*.txt
-include doc/html/*.png
-include doc/html/*.qhp
-include doc/html/*.qhcp
-include doc/html/Makefile
-include doc/po4a.conf doc/*.po doc/*.pot
-include doc/en/*.1 doc/en/*.5
-include doc/de/*.1 doc/de/*.5
-include doc/Makefile
-include doc/examples/*.sh doc/examples/*.bat doc/examples/*.py
-include doc/examples/linkcheckerrc_loginurl
-include scripts/*.sh scripts/*.py
-# tests
-include robots.txt
-include tests/configuration/data/*.ini
-recursive-include tests *.py *.result *.html *.ico *.txt *.zip *.asc *.css *.xhtml *.sqlite *.adr *.swf *.bin *.markdown *.pdf *.php *.plist *.wml *.xml
+include install-rpm.sh
+include linkchecker.freecode
+include .project
+include .pydevproject
+  doc/web/wokconfig
```

Comment:
Don't think it matters for linkchecker but N.B. something went wrong here.

warning: manifest_maker: MANIFEST.in, line 13: unknown action 'doc/web/wokconfig'

Reply:
Oops.